tests/PaymentGatewayConstructorTest.php

```diff
@@ -0,0 +1,231 @@
+<?php
+/**
+ * User: Becopay Team
+ * Version 0.0.1
+ * Date: 10/10/18
+ * Time: 10:36 AM
+ */
+
+namespace Test;
+
+use PHPUnit\Framework\TestCase;
+use Becopay\PaymentGateway;
+
+/**
+ * Class PaymentGatewayTest
+ *
+ * @package Test
+ */
+class PaymentGatewayConstructorTest extends TestCase
+{
+
+    private $dataSet = array(
+        //test constructor with correct data
+        array(
+            'apiUrl' => 'http://api.com',
+            'gatewayUrl' => 'http://gatwayurl',
+            'apiKey' => 'apikey',
+            'mobile' => '09100000000',
```

Comment:
mobile numbers remove

Reply:
this is just sample mobile  for test constructor validation parameter